web/webpack.config.js

```diff
@@ -88,47 +88,9 @@ module.exports = {
         use: "babel-loader",
         test: /\.(js|jsx)$/
       },
-      /* HACK: remove unwanted fonts from PatternFly's css */
-      {
-        test: /patternfly-4-cockpit.scss$/,
```

Comment:
Please, see the commit description.

Now I have doubts if we should start importing that file instead. Comments are welcome. 

Of course, we can continue working and change our mind later.

Reply:
I meant https://github.com/cockpit-project/cockpit/tree/267/pkg/lib/patternfly.

Reply:
Note, there [are more options](https://github.com/patternfly/patternfly/tree/main/src/patternfly) when importing a SCSS patternfly file, like just [patternfly.scss](https://github.com/patternfly/patternfly/blob/main/src/patternfly/patternfly.scss) but it includes too much for our current use.